webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx

```diff
@@ -221,11 +226,13 @@ my_model.load_state_dict(ckpt['models_state_dict'][0])`;
                 {action.text}
               </Button>
             ))}
-            <Dropdown menu={menu} trigger={['click']}>
-              <Button type="text">
-                <Icon name="overflow-horizontal" size="tiny" />
-              </Button>
-            </Dropdown>
+            {actions.length > 2 && (
```

Comment:
this should be more then 1 instead of 2?

Reply:
the actions should always include Download and Use in Notebook, is that wrong?

Reply:
depends on the screen size

https://user-images.githubusercontent.com/109113616/210443761-a2db53b8-d603-4908-bfa8-b318d0958a80.mov